packages/frontend-2/app.vue

```diff
@@ -36,6 +36,8 @@ watchAuthQueryString()
 await useMixpanelInitialization()
 </script>
 <style>
+@import 'https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap';
```

Comment:
@Mikehrn You shouldn't do this in app.vue, it's a change that should belong to the design system and thus be auto-applied anywhere where our tailwind theme is used. Not only that, but we don't want these fonts to be loaded from an external API, because then our app's performance becomes dependant on external resources that we can't control (e.g. what if googleapis.com goes down? or undergoes high load and becomes slow?).

tl;dr; These fonts should be actually downloaded into the `tailwind-theme` package, and then imported/applied by the tailwind theme plugin (or preset?), so that you don't need to do anything manually in FE2 or any other apps to ensure that the font is loaded.

Reply:
The reason why I went with the CDN is because in most cases it would be more performant when the user already has this file cached (which is with Inter a pretty high chance), in case it's not available it will fallback on the fonts we are currently using. We only added it to FE2 on purpose, to only affect the web app for now, but we will talk with Dim to check if we can just enable it everywhere :) If so I'll change it to the `tailwind-theme`

Reply:
@Mikehrn If it's only meant for FE2, then there shouldn't be any changes to the tailwind-theme package. That said, I don't see why we would want the font to differ between apps, if the theme/design system is the same.